README.md

```diff
@@ -14,7 +18,16 @@ in various formats.
 [![Latest Release][latest-release-badge]][latest-release-url]
 [![LGTM Score C++][lgtm-badge]][lgtm-url]
 
-## Synopsis
+[_Getting Started_](#getting-started) —
+[_Documentation_][docs] —
+[_Development_][contributing-url] —
+[_Chat_][chat-url] —
+[_Changelog_][changelog-url] —
+[_License and Scientific Use_](#license-and-scientific-use)
+
+
+Getting Started
+---
```

Comment:
Why the switch for header styles? We use preceding #'s everywhere else.

Reply:
That's just a proposal. We can keep the `##` if there is a strong opinion towards them. Personally,I just like that format better.

Reply:
The issue I see is that you cannot easily use h3 with this style. h1 is === and h2 is ---. No strong opinions from my side, though.

I'm dismissing my review since the requested changes were addressed.